VideoWeb/VideoWeb/ClientApp/src/app/shared/mic-visualiser/mic-visualiser.component.ts

```diff
@@ -96,6 +95,7 @@ export class MicVisualiserComponent implements AfterViewInit, OnDestroy, AfterVi
     fillMeter(feedback: number) {
         const canvasWidth = this.meterCanvas.nativeElement.scrollWidth;
         const canvasHeight = this.meterCanvas.nativeElement.scrollHeight;
+        this.canvasContext.fillStyle = this.fillColor;
```

Comment:
This was previously in this method. I had moved it to AfterViewInit to reduce the amount of times it would be set but it didn't seem to work. Reverting to what did work.